### PR TITLE
[gulpfile,css,xkit_preferences] Code style and appearance cleanup.

### PR DESCRIPTION
**Pull request by nepeat**
Finding what I found in the One Click Postage stylesheets made me check if the same style issues were in the other styles. They were in the other styles and I pain.

# Changes
* xkit_preferences
  * Use borders instead of images to separate columns.
* css
  * Remove excessive trailing newlines before and after braces.
  * Add trailing newlines at the end of every file.
  * Remove trailing tabs and whitespaces at each line.
  * Fix missing semicolons.
  * Remove rules/classes that are duplicated, empty, or do not show.
  * Add on some Firefox and generic vendor prefixing.
  * Fix some newlines to separate rules.
* js
  * Bump up versions of all the changed extensions. (I pain.)

# Visible Changes
### Before
Before
### After
Before

### CODE REVIEW

File: Extensions/activity_plus.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //* TITLE Activity+ **//
-//* VERSION 0.3 REV C **//
+//* VERSION 0.3.3 **//
```
Comment (nightpool) [inline Extensions/activity_plus.js]: not an update—"rev a == patch 1"

Comment (Wolvan) [inline Extensions/activity_plus.js]: No. Rev A => Patch 0

Comment (nepeat) [inline Extensions/activity_plus.js]: `Rev C => 2` right?

Comment (nightpool) [inline Extensions/activity_plus.js]: listen to @wolvan he wrote the code

Comment (Wolvan) [inline Extensions/activity_plus.js]: @nepeat you are correct. 0.3 REV C translates to 0.3.2

File: gulpfile.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /* jshint node: true */
-'use strict';
+'use strict'; // jshint ignore:line
```
Comment (nightpool) [inline gulpfile.js]: would prefer something more targeted. Also, do you have issues with this? I don't think I've ever seen warnings about it....

(are you using gulp for linting?)

Comment (nepeat) [inline gulpfile.js]: gulp was being used for the server mode but jshint ignored the node mode comment above that line. that line being ignored making the server not run and build the dust unless the use strict line was specifically excluded

Comment (0xazure) [inline gulpfile.js]: @nepeat can you give a step-by-step to reproduce?  I've run both `gulp lint` and `gulp server` and haven't run into to issue you describe.

:-1: on this line change, it's inelegant and shouldn't be necessary.

**Comment by 0xazure** [comment]
Personally, I'm not sure I'm a fan of having all of these changes in a single commit/PR, since the scope is large and there's a lot of things to check, making it easy to miss things.

Separating out style changes (newlines, indentation, etc) from actual code changes (changing values) would help to make it clearer which changes are just style and which ones require manual testing verification.

**Comment by nightpool** [comment]
The blocker for me on this issue is that we should probably have some sort of css linting in the long run. On the other hand, I spent like 30 minutes playing with csslinter and it was absolutely terrible. Even aside from all of the terrible rules (no adjoining classes?? with the only justification being "it breaks on IE6"???), it wouldn't even parse `.controls_section:not(.controls_section.follow_list.recommended_tumblelogs)`.

does anyone know an alternative css linter?

**Comment by nightpool** [comment]
in general, :-1: from me to style changes that aren't associated with a lint. Either we enforce a consistent rule programmatically through linting, or we just don't worry about it (I'm fine with either, and probably prefer the latter for css TBQH)

**Comment by 0xazure** [comment]
Agreed with @nightpool.

Linting discussion can be merged into #209, or we can open a separate issue for css-specific rules.  I think [CSSLint](https://github.com/CSSLint/csslint) is still our best bet since it's more fully-featured, but we probably need to heavily reconfigure the defaults.  My understanding was we hadn't started enforcing css linting because it would result in another touch-everything merge to align our existing code with the linter to get CI to pass.

File: gulpfile.js (hunk 2 above)
Comment (nepeat) [inline gulpfile.js]: No idea what is going on with my setup but this is how the issue happens when the line is omitted.

https://gist.github.com/nepeat/0a3f934ba7beb040bca5

# Versions
* Gulp - 3.9.0
* gulp-jshint - 1.11.2
* jshint - v2.7.0

Comment (Wolvan) [inline gulpfile.js]: Your problem is jshint 2.7.0 afaik. Do `gulp clean:modules` and then `npm i`

Comment (nepeat) [inline gulpfile.js]: Looks like doing a module purge and reinstalling all of them worked. Going to rebase this change out of the fork.

**Pull request closed without merging**